library/network/src/lib/y2firewall/firewalld/interface.rb

```diff
@@ -96,6 +96,15 @@ def zone
         fw.zones.find { |z| z.interfaces.include?(name) }
       end
 
+      # Assign the interface to the given zone
+      #
+      # @param zone_name [String] the name of the zone to be assigned to
+      def zone=(zone_name)
+        fw.zones.map { |z| z.remove_interface(name) if z.interfaces.include?(name) }
```

Comment:
Why `.map`? I think `.each` could be enough, isn't it?

Reply:
Yep, it is.

As it was added here:

https://github.com/yast/yast-yast2/pull/896/files#diff-5f18f6aba33e047bd40791544760f288R144

Probably the inertia.. :smile: